cfgov/jinja2/v1/_includes/atoms/large_radio.html

```diff
@@ -0,0 +1,59 @@
+{# ==========================================================================
+
+   large_radio.render()
+
+   ==========================================================================
+
+   Builds radio button fields with a large hit target.
+
+   settings (required):
+
+   settings.legend:   Name of the field
+
+   settings.fields:   An array of all the radio button labels.
+
+   settings.size:     The size of the fields. Defaults to 1-2.
+                      Possible options: 1, 1-2, 1-3
+
+   options (optional):
+
+   options.required:  Whether the field is required. Defaults to false.
+
+   options.disabled:  Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro render(settings, options) -%}
+    {% from 'macros/util/format/url.html' import slugify as slugify %}
+    {% set id_slug = slugify( settings.legend ) %}
+    <fieldset class="u-reset">
+        <div class="form-l_col
+                    form-l_col-1
+                    u-mb0">
```

Comment:
I haven't touched `form-l` because that's part of the forms and I want to keep this focus on the the buttons and checkboxes. I'll revisit when I get to creating the form macro/includes.